packages/salesforcedx-utils-vscode/src/helpers/paths.ts

```diff
@@ -21,7 +21,7 @@ export const DEBUG = 'debug';
 export const LOGS = 'logs';
 export const APEX_DB = 'apex.db';
 export const LWC = 'lwc';
-export const SFDX_CONFIG_FILE = 'sfdx-config.json';
+export const SF_CONFIG_FILE = 'config.json';
```

Comment:
I'm curious, why did you decide to call it `config.json` instead of `sf-config.json`?